Madness/Map.swift

```diff
@@ -47,5 +52,12 @@ infix operator <^> {
 	precedence 130
 }
 
+/// Replace operator.
+infix operator <^ {
```

Comment:
It could also be useful to add `^>` I suppose, but because Haskell defines only `<$` I haven't added it here (for now).